### PR TITLE
Made it more modern.

### PR DESCRIPTION
**Pull request by alexindigo**
Brought dependencies up to date and fixed syntax error in `mc.js`.

### CODE REVIEW

File: mc.js [hunk 1]
```diff
@@ -207,8 +207,8 @@ mix(Mc.prototype,{
       get value(){
         return buf;
       },
-      set value(val,mode){
-        buf = mode=='append'?buf+val:(mode=='prepend'?val+buf:val);
+      set value(val){
```
Comment (soldair) [inline mc.js:210]: this seems to remove support for memcache append and prepend commands. is this intentional or handled in a different way i dont see?

Comment (soldair) [inline mc.js:210]: ha i guess it never had support for it. thanks!

**Comment by soldair** [comment]
published to npm v1.0.0

File: mc.js (hunk 1 above)
Comment (alexindigo) [inline mc.js:210]: Yeah, somewhere between `0.9` and `0.12` nodejs started to care about ES syntax :)

**Comment by alexindigo** [comment]
Cool, thanks a lot.

**Pull request merged**